src/main/java/com/quantumsoft/qupathcloud/dao/spec/InstancesPathBuilder.java

```diff
@@ -15,38 +15,58 @@
 
 package com.quantumsoft.qupathcloud.dao.spec;
 
+import static com.quantumsoft.qupathcloud.dao.Constants.DATASETS;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_STORES;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_WEB;
+import static com.quantumsoft.qupathcloud.dao.Constants.INSTANCES;
+import static com.quantumsoft.qupathcloud.dao.Constants.LOCATIONS;
+import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
+import static com.quantumsoft.qupathcloud.dao.Constants.SERIES;
+import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
+import static com.quantumsoft.qupathcloud.dao.Constants.STUDIES;
+
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
 
-import static com.quantumsoft.qupathcloud.dao.Constants.*;
+/**
+ * InstancesPathBuilder to build a path for access to Instances in chosen Project, Location,
+ * Dataset, DICOM Store, Study and Series. If Study and Series are null then will be built a path
+ * for all Instances in a chosen Project, Location, Dataset, and DICOM Store.
+ */
```

Comment:
InstancesPathBuilder builds a path to access Instances in the specified Project, Location,
 Dataset, DICOM Store, Study and Series. If the Study and Series are null then the path will be built
 for all Instances in a chosen Project, Location, Dataset, and DICOM Store.